docs/release/release-1.4.rst

```diff
@@ -0,0 +1,12 @@
+===========
+Release 1.4
+===========
+
+This release works for Presto versions 0.116-0.148.
+
+* Add package uninstall support
+* Add --nodeps option to indicate if the server install/uninstall should ignore dependencies
+* Fix config files to be owned by the presto user and not accessible to other users
+* Update and add more Presto configuration defaults
+* Fix bug to use JAVA 8 for server upgrade
```

Comment:
suggest: "Use proper Java version for server upgrade"

(that's what it was right?)